corgi/collectors/brew.py

```diff
@@ -768,3 +770,110 @@ def brew_rpm_lookup(self, rpms) -> tuple:
         with self.koji_session.multicall() as multicall:
             find_build_id_calls = tuple((rpm, multicall.getRPM(rpm)) for rpm in rpms)
         return find_build_id_calls
+
+    def sans_epoch(self, srpm) -> str:
+        name, version, release = self.split_nvr(srpm)
+        version_parts = version.split(":")
+        if len(version_parts) > 1:
+            srpm = f"{name}-{version_parts[1]}-{release}"
+        return srpm
+
+    def module_key_to_nvr(self, module_key) -> str:
+        module_parts = module_key.split(":")
+        return f"{module_parts[0]}-{module_parts[1]}-{module_parts[2]}.{module_parts[3]}"
```

Comment:
Could you add a header comment explaining why these are needed? I understand what the first one does, but I don't quite get the second one. Probably because I don't know what a module_key is or what module_parts it contains. A special kind of NVR?

Reply:
Will do.